src/OmniSharp.Roslyn.CSharp/Services/Types/V2/TypeLookUp.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Mef;
+using OmniSharp.Models.v2.TypeLookUp;
+using OmniSharp.Options;
+using OmniSharp.Roslyn.CSharp.Services.Documentation;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Types.V2
+{
+    [OmniSharpHandler(OmniSharpEndpoints.V2.TypeLookup, LanguageNames.CSharp)]
+    public class TypeLookupService : IRequestHandler<TypeLookupRequest, TypeLookupResponse>
+    {
+        private readonly FormattingOptions _formattingOptions;
+        private readonly OmniSharpWorkspace _workspace;
+        private static readonly SymbolDisplayFormat DefaultFormat = SymbolDisplayFormat.FullyQualifiedFormat.
```

Comment:
Where did this come from?

Reply:
@rchande here is the background https://github.com/OmniSharp/omnisharp-roslyn/pull/665